content/en/synthetics/mobile_app_testing/mobile_app_tests/steps.md

```diff
@@ -172,11 +172,114 @@ If you know that a page or page element takes more than 60 seconds to load, you
+   * **HTTP Basic Auth**: Add HTTP basic authentication credentials.
+   * **Digest Auth**: Add Digest authentication credentials. 
+   * **AWS Signature**: Add AWS Access Key ID and Secret Access Key.
+   * **NTLM**: Add NTLM authentication credentials. Support both NTLMv2 and NTLMv1.
+   * **OAuth 2.0**: Select a Grant Type (Client credentials, or Resource owner password).
+
+   {{% /tab %}}
+
+   {{% tab "Query Parameters" %}}
+
+   * **Encode parameters**: Add the name and value of query parameters that require encoding. 
+
+   {{% /tab %}}
+
+   {{% tab "Request Body" %}}
+
+   * **Body type**: Select the type of the request body (`text/plain`, `application/json`, `text/xml`, `text/html`, `application/x-www-form-urlencoded`, `GraphQL`, or `None`) you want to add to your HTTP request.
+   * **Request body**: Add the content of your HTTP request body. The request body is limited to a maximum size of 50 kilobytes.
+
+   {{% /tab %}}
+
+   {{% tab "Proxy" %}}
+
+   * **Proxy URL**: Specify the URL of the proxy the HTTP request should go through (`http://<YOUR_USER>:<YOUR_PWD>@<YOUR_IP>:<YOUR_PORT>`).
+   * **Proxy Header**: Add headers to include in the HTTP request to the proxy.
+
+   {{% /tab %}}
+  
+   {{% tab "Privacy" %}}
+
+   * **Do not save response body**: Select this option to prevent the response body from being saved at runtime. This helps ensure no sensitive data is displayed in your test results, but it can make failure troubleshooting more difficult. For full security recommendations, see [Synthetic Monitoring Data Security][1].
+
+[1]: /data_security/synthetics
+   {{% /tab %}}
+
+   {{< /tabs >}}
+   </br>
+3. Click **Send** to try out the request configuration. A response preview appears.
+
+{{< img src="mobile_app_testing/test_steps/http_mobile_request.png" alt="Make HTTP Request" style="width:80%;" >}}
+
+##### Add assertions
+
+Assertions define what an expected test result is. After you click **Send**, basic assertions on `status code`, `response time`, and `header` `content-type` are added based on the test response. Assertions are optional for HTTP steps in browser tests.
+
+| Type          | Operator                                                                                               | Value type                                                      |
+|---------------|--------------------------------------------------------------------------------------------------------|----------------------------------------------------------------|
+| body          | `contains`, `does not contain`, `is`, `is not`, <br> `matches`, `does not match`, <br> [`jsonpath`][11], [`xpath`][12] | _String_ <br> _[Regex][13]_ <br> _String_, _[Regex][13]_ |
+| header        | `contains`, `does not contain`, `is`, `is not`, <br> `matches`, `does not match`                       | _String_ <br> _[Regex][13]_                                      |
```

Comment:
These links are broken - it looks like all you need to do is add in the link references at the bottom of the page and it'll fix it 🙂

Reply:
Same notes as above about formatting the items in the Type column and the breaks! Just figured that if you want to switch up the breaks in the other table that you might just want to duplicate it here.

Reply:
let me check live, if they are broken in live, that is an oversight. thanks for catching.

Reply:
fixed!